pkg/apis/config/v1alpha4/types.go

```diff
@@ -189,6 +189,8 @@ type Networking struct {
 	// KubeProxyMode defines if kube-proxy should operate in iptables or ipvs mode
 	// Defaults to 'iptables' mode
 	KubeProxyMode ProxyMode `yaml:"kubeProxyMode,omitempty" json:"kubeProxyMode,omitempty"`
+	// DNSSearch defines the DNS search domain to use for nodes. If not set, this will be inherited from the host.
+	DNSSearch *[]string `yaml:"dnsSearch,omitempty" json:"dnsSearch,omitempty"`
```

Comment:
do we need a pointer ? is not the slice going to be nil if is empty?

Reply:
Intent was to allow `dnsSearch: []` to set it to empty, but I am open to suggestions.. I don't love it either.

Reply:
got it https://github.com/kubernetes-sigs/kind/pull/3098/files#r1106429110